### ISSUE TITLE
Enhance the accessibility of `MultiSelect` (@yamada-ui/select) component

### ISSUE THREAD

**Issue by koralle** [yamada-ui/yamada-ui#2059]
### 1. Accessibility roles

![Image](https://github.com/yamada-ui/yamada-ui/assets/33865215/2de94fc7-211d-4b20-9a1d-58bd2b5ba246)

### 2. `aria-controls`

* [x] An element with the `combobox` role must have in its `aria-controls` attribute the ID of the corresponding popup element (the element with the `listbox` role).

### 2. `aria-activedescendant`

* [x] When any element with the `option` role within an element with the `listbox` role receives focus, you need to set the ID of that element to the `aria-activedescendant` attribute of the element with the `combobox` role.

### 3. `aria-selected`

* [x] Set `aria-selected="true"` on the selected elements with the `option` role. For the elements with the option role that are not selected but can be selected, set `aria-selected="false"`. If an element cannot be selected, do not set the `aria-selected` attribute at all.

### 4. `aria-multiselectable`

* [x] You need to set `aria-multiselectable="true"` on the element with the `listbox` role to inform assistive technologies that multiple selection is possible.

## Keyboard Interaction

* [ ] When `Alt` + `Up Arrow` is pressed while the popup is displayed...
  * [ ] If an element within the popup is focused, return the focus to the element with the `combobox` role.
  * [ ] Close the popup.

## References

* [ ] [MultiSelect - Yamada UI](https://yamada-ui.com/components/forms/multi-select)

### Problem Statement/Justification

We should prioritize web accessibility to ensure our UI components are usable by everyone, including people with disabilities. This inclusivity enhances user experience and complies with legal standards, broadening Yamada UI's user base.

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by zonoryo17** [comment]
I'll work on this.

**Comment by zonoryo17** [comment]
I had assigned it, but I will remove it once as I have been holding it too long.

**Comment by chida09** [comment]
>  An element with the combobox role must have in its aria-controls attribute the ID of the corresponding popup element (the element with the listbox role).

Already supported.
<img width="1323" alt="スクリーンショット 2024-09-22 8 24 22" src="https://github.com/user-attachments/assets/885d8731-4847-4659-9da5-9c92686e9fae">

**Comment by chida09** [comment]
>3. aria-selected
 Set aria-selected="true" on the selected elements with the option role. For the elements with the option role that are not selected but can be selected, set aria-selected="false". If an element cannot be selected, do not set the aria-selected attribute at all.

I determined that no action was necessary because although aria-selected does not exist, aria-checked is already present.
Therefore, no action is required.

ref: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected
>Don't specify both aria-selected and aria-checked on option elements contained by the same listbox unless the meaning and purpose of aria-selected is different from the meaning and purpose of aria-checked in the user interface, the meaning and purpose of each state apparent, and the UI provides separate methods for controlling each state.

**Comment by chida09** [comment]
>2. aria-activedescendant
 When any element with the option role within an element with the listbox role receives focus, you need to set the ID of that element to the aria-activedescendant attribute of the element with the combobox role.

Already supported.
<img width="1209" alt="スクリーンショット 2024-09-22 10 20 48" src="https://github.com/user-attachments/assets/16152493-9554-4da3-90db-b75320aa277a">

**Comment by uraaaa24** [comment]
@hirotomoyamada 
> When Alt + Up Arrow is pressed while the popup is displayed...
 If an element within the popup is focused, return the focus to the element with the combobox role.
 Close the popup.
 
 This one is optional according to the W3C documentation, should I implement it?
